tutorials/iot-ds-3-create-ui/iot-ds-3-create-ui.md

```diff
@@ -4,7 +4,7 @@ description: Create a decision support application using the SAP IoT extension f
 auto_validation: true
 time: 30
 primary_tag: topic>internet-of-things
-tags: [ tutorial>beginner, tutorial>license, topic>internet-of-things, topic>cloud, topic>sapui5, products>sap-leonardo-iot, products>sap-edge-services, products>sap-cloud-platform-internet-of-things, products>sap-cloud-platform, products>sap-web-ide ]
+tags: [ tutorial>beginner, tutorial>license, topic>internet-of-things, topic>cloud, topic>sapui5, products>sap-edge-services, products>sap-cloud-platform-internet-of-things, products>sap-business-technology-platform, products>sap-web-ide ]
```

Comment:
products>sap-cloud-platform-internet-of-things -> wrong tag

Reply:
I have removed the wrong tags and replaced the usage of we/our with you/your